packages/scss/src/theming/components/_button.theme.scss

```diff
@@ -30,6 +30,9 @@ $button: (
 		color: black,
 	),
 
+	radio: _theme("palettes.primary"),

```

Comment:
If we use a single class like button-group-input, we would use only one variable